README.md

```diff
@@ -131,14 +123,10 @@ Valid properties for landmark specification referenced by `landmark_url`:
 
 * RID - (string) The ID of the Landmark.
 * mesh - (string) The ID of the Mesh this landmark points
+* anatomy - (string) Name of the anatomy this landmark points
+* anatomy_id - (string) ID for the anatomy. Used with `anatomy_url_fragment` for constructing a link back to the anatomy.
+* label - (string) Can be used instead of `anatomy` as an alternative description.
```

Comment:
"as an alternative for the display label for the landmark, if the `anatomy` is not given."